src/google_form_to_engagement_db/google_form_to_engagement_db.py

```diff
@@ -290,15 +311,42 @@ def _sync_google_form_to_engagement_db(google_form_client, engagement_db, form_c
             sync_stats.add_event(GoogleFormSyncEvents.READ_ANSWER_FROM_RESPONSE)
             if answer["questionId"] not in question_id_to_engagement_db_dataset:
                 log.info(f"This answer is to question {answer['questionId']}, which isn't configured in this sync")
-                continue
+                continue 
 
-            message = _form_answer_to_engagement_db_message(
+            question_id_to_engagement_db_message[answer["questionId"]] = _form_answer_to_engagement_db_message(
                 answer, form_config.form_id, response, participant_uuid, question_id_to_engagement_db_dataset
             )
-            message_origin_details = {
+            question_id_to_message_origin_details[answer["questionId"]] = {
                 "formId": form_config.form_id,
                 "answer": answer,
             }
+
+        for question_config in form_config.question_configurations:
+            assert len(question_config.question_titles) > 0
+            if len(question_config.question_titles) == 1:
+                question_id = question_title_to_question_id[question_config.question_titles[0]]
+                if question_id not in question_id_to_engagement_db_message:
+                    continue
+                message = question_id_to_engagement_db_message[question_id]
+                message_origin_details = question_id_to_message_origin_details[question_id]
+            else:
+                messages = []
+                messages_origin_details = []
+                for question_title in question_config.question_titles:
+                    question_id = question_title_to_question_id[question_title]
+                    if question_id not in question_id_to_engagement_db_message:
+                        continue
+                    messages.append(question_id_to_engagement_db_message[question_id])
+                    messages_origin_details.append(question_id_to_message_origin_details[question_id])
+                if not len(messages) > 0:
+                    continue
+                elif len(messages) == 1:
+                    [message], [message_origin_details] = messages, messages_origin_details
+                else:
+                    # TODO: Implement the function below that merges engagement db messages including their origin details
+                    # message, message_origin_details = _merge_engagement_db_messages(messages, messages_origin_details, question_config.answers_delimeter)
```

Comment:
This is a big todo - I think you should crash here instead for now

Reply:
Added a `continue` statement so that you can be able to test for other cases.

Reply:
Given this will be addressed very soon in #233 this is fine for now